### PR TITLE
kv: move TxnScope into kv

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?

Issue Number: 
https://github.com/pingcap/tidb/issues/22513

Problem Summary: 
`oracle.TxnScope` is mainly for handling txn scope session variable in TiDB. All tikv / pd needs is just a string.

### What is changed and how it works?

What's Changed:
- move `oracle.TxnScope` -> `kv.TxnScopeVar`
- move `oracle.LocalTxnScope` -> `kv.LocalTxnScope`
- define `kv.GlobalTxnScope = oracle.GlobalTxnScope`
- use `kv.GlobalTxnScope`, `kv.LocalTxnScope` everywhere expect for `store/tikv`.

### Check List 

Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/oracle/oracle.go [hunk 1]
```diff
@@ -45,57 +44,11 @@ type Future interface {
...
-		varValue: varValue,
-		txnScope: txnScope,
-	}
-}
-
 const (
 	physicalShiftBits = 18
 	logicalBits       = (1 << physicalShiftBits) - 1
 	// GlobalTxnScope is the default transaction scope for a Oracle service.
 	GlobalTxnScope = "global"
```
Comment (JmPotato) [inline store/tikv/oracle/oracle.go:51]: Should we remove these two lines too?

Comment (JmPotato) [inline store/tikv/oracle/oracle.go:51]: Should we remove these two lines too?

Comment (disksing) [reply store/tikv/oracle/oracle.go:51]: It is a special constant exposed by PD and `store/tikv` uses it too.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by Yisaer — APPROVED** [review]

**Pull request merged**